src/app/src/components/Submissions/Detail/Map.js

```diff
@@ -0,0 +1,78 @@
+import { Box, Button, Flex, HStack, Icon, Text } from '@chakra-ui/react';
+import { ChatAltIcon, DownloadIcon, MailIcon } from '@heroicons/react/outline';
+import { InformationCircleIcon } from '@heroicons/react/solid';
+import { MapContainer } from 'react-leaflet';
+
+import { DefaultBasemap } from '../../Layers/Basemaps';
+import MapPanes from '../../MapPanes';
+
+import { MAP_CENTER, MAP_INITIAL_ZOOM } from '../../../constants';
+
+export default function Map({ submission }) {
+    return (
+        <>
+            <MapContainer
+                center={MAP_CENTER}
+                zoom={MAP_INITIAL_ZOOM}
+                zoomControl={false}
+                style={{ height: '100%' }}
+            >
+                <MapPanes>
+                    <DefaultBasemap />
+                    <MapButtons submission={submission} />
+                    <SubmissionStatusBar submission={submission} />
+                </MapPanes>
+            </MapContainer>
+        </>
+    );
+}
+
+function MapButtons({ submission }) {
+    return (
+        <Box position='absolute' zIndex={1000} top={22} w='100%'>
+            <Flex justify='space-evenly'>
```

Comment:
I think we may add a max-width to that page so that it doesn't get too wide on wider screens. The buttons look centered / evenly spaced to me.

I think this is good enough for now, we can revisit if we need to.